<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Luiss-MacBook-Air.local" failures="1" tests="3" name="Test Report 2" time="19.261" errors="0" timestamp="2024-03-26T21:10:40 PET">
  <testcase classname="com.reports.tests.ReportTest2" name="getTitle2" time="0.008"/>
  <testcase classname="com.reports.tests.ReportTest2" name="skyTest2" time="0.005">
    <skipped/>
  </testcase> <!-- skyTest2 -->
  <testcase classname="com.reports.tests.ReportTest2" name="getLogo2" time="0.035">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
at com.reports.tests.ReportTest2.getLogo2(ReportTest2.java:43)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- getLogo2 -->
</testsuite> <!-- Test Report 2 -->
